Classes/Private/Manager/Database/OCTRealmManager.m

```diff
@@ -258,6 +260,25 @@ - (void)removeChatWithAllMessages:(OCTChat *)chat
     });
 }
 
+- (void)removeAllCalls
+{
+    RLMResults *calls = [OCTCall objectsInRealm:self.realm where:nil];
+
+    DDLogInfo(@"OCTRealmManager: removing %lu calls", calls.count);
+
+    RBQRealmChangeLogger *logger = [self logger];
+
+    for (OCTCall *call in calls) {
+        OCTMessageCallEvent event = (call.status != OCTCallStatusActive) ? OCTMessageCallEventUnanswered : OCTMessageCallEventAnswered;
```

Comment:
Even better it would be to use switch here. This way in case if new value will be added to enum compiler will give you an error.

```
switch (call.status) {
    case OCTCallStatusRinging:
    case OCTCallStatusDialing:
        event = OCTMessageCallEventUnanswered;
        break;
    case OCTCallStatusActive:
        event = OCTMessageCallEventAnswered;
        break;
}
```